spec/travis-actionmailer.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -v -e
+
+rm -rf rails
+git clone --depth=1 https://github.com/rails/rails.git -b master
+cd rails
+echo "" >> Gemfile
+echo "gem 'mail', path: '../'" >> Gemfile
+rm -rf Gemfile.lock
+bundle install
```

Comment:
`bundle update --source mail` may be sufficient, rather than removing the lock and reinstalling